main/test/ca/uwaterloo/flix/library/TestBigInt.flix

```diff
@@ -745,4 +747,26 @@ namespace TestBigInt {
     @test
     def clampToFloat6407(): Bool = BigInt.clampToFloat64(-8982741234134123419879712341ii, -100.0f64, 100.0f64) == -100.0f64
 
+    /////////////////////////////////////////////////////////////////////////////
```

Comment:
I think these being tests for the basic operations-- we should just move them to the top of the test file. So its order goes:

- tests of primitives (in the order of SemanticOperator)
- tests of whatever is in e.g. Int32
-